### PR TITLE
Update unsorted.m3u

### PR DESCRIPTION
**Pull request by themak1985**

### CODE REVIEW

**Comment by themak1985** [comment]
The three channels remaining in unsorted are internet only channels - I could place them in a country based on whois data for the domains, but wasn't sure if that was best approach. I wouldn't use server location, as that can mean anything.

**Comment by LaneSh4d0w** [comment]
You did what I wanted to do. Let me do some additional research on those channels, and come back to you if I see some anomalies.

File: channels/us.m3u [hunk 1]
```diff
@@ -1071,3 +1071,13 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="",Z Living
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/tracks-v1a1/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US;UK" tvg-language="English" tvg-logo="https://i.imgur.com/pggeczF.jpg" group-title="Sport",World of Free Sports
+https://a.jsrdn.com/broadcast/ab14783a09/+0000/high/c.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:1075]: Again, cannot really tell if this channel is really from the US...

Found one YouTube channel with the same name that claims to be German.

I also found a UK channel called FreeSports that sometimes broadcasts a program called "World of Free Sports"...

Could be someone that compilated all of those and had put it into a stream.

File: channels/us.m3u [hunk 2]
```diff
@@ -1071,3 +1071,13 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/DPls9e7.png" group-title="",The Pet Collective
+https://dai2.xumo.com/amagi_hls_data_xumo1212A-petcollective/CDN/1280x720_5000000/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US;UK" tvg-language="English" tvg-logo="https://i.imgur.com/5uIXfol.jpg" group-title="",The Boat Show
+https://a.jsrdn.com/broadcast/22706/+0000/hi/c.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US;UK" tvg-language="English" tvg-logo="" group-title="",The Monarch Channel
+https://a.jsrdn.com/broadcast/0c9a09c94c/+0000/c.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:1081]: Well, lots of channels seems to have the same pattern. Might be fan streams then.

**Review by BellezaEmporium — COMMENTED** [review]
Few things to say on this.

**Comment by mikcc123** [comment]
感謝這位大神的貢獻，希望大神可以增加VIUTV直連的頻道，這樣可以雙語切換，現在這個只能單語言比較遺憾，還有希望增加東森電影和台視這兩個頻道。

**Comment by mikcc123** [comment]
Hong Kong’s VIUTV channel often fails to broadcast. It is recommended to do a web broadcast like J2 and Jade.

**Comment by mikcc123** [comment]
> Hong Kong’s VIUTV channel often fails to broadcast. It is recommended to do a web broadcast like J2 and TVB.

香港的VIUTV頻道 經常播放失敗建議和J2和翡翠臺一樣做個網頁轉播

**Review by freearhey — APPROVED** [review]

**Comment by mikcc123** [comment]
ViuTV channel malfunction, please update

**Comment by LaneSh4d0w** [comment]
> 
> 
> 感謝這位大神的貢獻，希望大神可以增加VIUTV直連的頻道，這樣可以雙語切換，現在這個只能單語言比較遺憾，還有希望增加東森電影和台視這兩個頻道。

This is the wrong section to talk about channels that are not treated in this PR.

Please make a channel request on the "Chinese Channel Requests" issue that is available.

File: channels/us.m3u (hunk 1 above)
Comment (themak1985) [reply channels/us.m3u:1075]: this is a distro.tv channel. Distro.tv is based in US and targets its channels to UK too.

File: channels/us.m3u (hunk 2 above)
Comment (themak1985) [reply channels/us.m3u:1081]: another distro.tv channel

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:1075]: Indeed.

File: channels/de.m3u [hunk 3]
```diff
@@ -208,3 +208,5 @@ https://srunserding-lh.akamaihd.net/i/visualradio_ud@197013/master.m3u8
 https://wdr_fs-lh.akamaihd.net/i/wdrfs_weltweit@112033/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/smyvTHD.jpg" group-title="Sport",World Of Free Sports TV
 https://a.jsrdn.com/broadcast/22723/+0000/hi/c.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="DE" tvg-language="German" tvg-logo="" group-title="Movies",ID HD (JKT)
```
[the remark this answers is not in the document]
Comment (themak1985) [reply channels/de.m3u:211]: it is a german language channel, everytime I have checked the stream, its either german language programming or german subtitles. I feel comfortable placing in germany now, and moving in future if need be.

File: channels/it.m3u [hunk 4]
```diff
@@ -222,3 +222,5 @@ http://wms.shared.streamshow.it:1935/veratv/veratv/playlist.m3u8
 https://content.uplynk.com/channel/36953f5b6546464590d2fcd954bc89cf.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IT" tvg-language="Italian" tvg-logo="https://www.dinamobasket.com/sites/default/files/videolina2012_tr.png" group-title="",Videolina (Sardegna) (404p)
 http://livestreaming.videolina.it/live/Videolina/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IT" tvg-language="English" tvg-logo="" group-title="Music",Radio Reload
```
[the remark this answers is not in the document]
Comment (themak1985) [reply channels/it.m3u]: right, its from cyprus.

File: channels/unsorted.m3u [hunk 5]
```diff
@@ -1,29 +1,7 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="English" tvg-logo="https://i.imgur.com/HL7fwzt.png" group-title="Sport",Adventure Sports TV (360p)
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="INT" tvg-language="English" tvg-logo="" group-title="Music",World Street Music
+https://59d39900ebfb8.streamlock.net/streetmusic/streetmusic/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="INT" tvg-language="English" tvg-logo="https://i.imgur.com/HL7fwzt.png" group-title="Sport",Adventure Sports TV (360p)
 https://gizmeon.s.llnwi.net/channellivev3/live/master.m3u8?channel=275
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="English" tvg-logo="https://i.imgur.com/HL7fwzt.png" group-title="",Afrobeats
-https://stream.ecable.tv/afrobeats/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="English" tvg-logo="https://i.imgur.com/2msWdzQ.jpg" group-title="Sport",Horizon Sports
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="INT" tvg-language="English" tvg-logo="https://i.imgur.com/2msWdzQ.jpg" group-title="Sport",Horizon Sports
```
[the remark this answers is not in the document]
Comment (themak1985) [reply channels/unsorted.m3u:6]: distro.tv is based in US - UK is also one of their markets, I feel comfortable listing as US:UK and basing in US.

File: channels/uk.m3u [hunk 6]
```diff
@@ -111,3 +111,5 @@ http://csm-e.tm.yospace.com/csm/extlive/boxplus01,thebox-desktop.m3u8?yo.up=http
 https://abr.de1se01.v2beat.live/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Ll6GlqY.png" group-title="Music",V2Beat TV (720p)
 https://de1se01.v2beat.live/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/HL7fwzt.png" group-title="",Afrobeats 247
+https://stream.ecable.tv/afrobeats/tracks-v1a1/mono.m3u8
```
[the remark this answers is not in the document]
Comment (themak1985) [reply channels/uk.m3u:115]: ecable created this channel AFAIK, ecable is an OTT provider in the UK that targets the African Diaspora in the UK.

**Pull request merged**